### DISCUSSION TITLE
FF dreamer stuck on FLashforge splashscreen after upgrade of firmware

### DISCUSSION THREAD

**Discussion by vncntv** [moonglow/FlashForge_Marlin#162] [Q&A] [answered]
Hello,

I have been using Moonglow's marlin on my dreamer for over a year (I used one of moonglows preconfigured installs). I recently upgraded to TMC2208 drivers and had to reverse the step direction, so I figured i would change the config.h file, compile and upload.

I am using VSCode and platformio, and started with the Flashforge marlin repo. I changed what I assume to be the relevant settings in the config.h file, adjusted the homing position and bed zero, and changed the stepper direction.

Everything compiles without issue, and I was able to use FF_fw_writer.exe to connect to my printer and upload firmware. The printer seems to register the firmware because after uploading the TFT display shows a progress bar slowly filling and it says 'File upload success' or something like that.

However this screen does not go away. Even when rebooting / resetting the printer it stays stuck on the flashforge splash screen.
Does anybody know how I can fix this?

I attached my config file as a txt for anyone who is curious.

Thanks in advance.

[Config.txt](https://github.com/moonglow/FlashForge_Marlin/files/13762336/Config.txt)

**Comment by GrandWaz00** [comment, marked as answer]
Did you encrypt the binary with ff_fw_tool.exe ?

Reminder: if you use my binary ( https://github.com/moonglow/FlashForge_Marlin/discussions/139 ) then you MUST specify the Flashforge encryption key on the command line, per https://github.com/moonglow/flashforge_firmware_tool

**Reply by vncntv** [reply]
Ah yes that did the trick! Thank you very much.

**Comment by GrandWaz00** [comment]
Your recovery from this oversight was A LOT quicker than mine! I am really happy it worked out for you!
